docs/proposals/kubelet-eviction.md

```diff
@@ -75,9 +75,12 @@ The `kubelet` will support the ability to trigger eviction decisions on the foll
 |------------------|---------------------------------------------------------------------------------|
 | memory.available | memory.available := node.status.capacity[memory] - node.stats.memory.workingSet |
 | nodefs.available   | nodefs.available := node.stats.fs.available |
-| nodefs.inodesFree | nodefs.inodesFree := node.fs.inodesFree |
+| nodefs.inodesFree | nodefs.inodesFree := node.stats.fs.inodesFree |
 | imagefs.available | imagefs.available := node.stats.runtime.imagefs.available |
-| imagefs.inodesFree | imagefs.inodesFree := node.runtime.imageFs.inodesFree |
+| imagefs.inodesFree | imagefs.inodesFree := node.stats.runtime.imagefs.inodesFree |
+
+Each of the above signals support both literal and percentage based values.  The percentage based values
```

Comment:
"Each of the above signals supports"

Can both literal and percentage be specified for a signal at the same time? From what I read below, this seems to be an unsupported usage. Please make it explicit if my understanding is right.